app/components/search/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+
+import { eligibilitiesMapping, categoriesMapping } from 'utils/refinementMappings';
+import ClearAllFilters from 'components/search/Refinements/ClearAllFilters';
+import OpenNowFilter from 'components/search/Refinements/OpenNowFilter';
+import RefinementListFilter from 'components/search/Refinements/RefinementListFilter';
+import FacetRefinementList from 'components/search/Refinements/FacetRefinementList';
+
+import filtersIcon from 'assets/img/filters-icon.png';
+import styles from './Sidebar.module.scss';
+
+const Sidebar = ({
+  isSearchResultsPage, eligibilities = [], subcategories = [], subcategoryNames = [],
+}: {
+  isSearchResultsPage: boolean;
+  eligibilities?: Array<object>;
+  subcategories?: Array<object>;
+  subcategoryNames?: Array<string>;
```

Comment:
Nit: For simple array types, let's use the shorter syntax: `ItemType[]`.

```suggestion
  eligibilities?: object[];
  subcategories?: object[];
  subcategoryNames?: string[];
```

Reply:
Got it. Good call.